.github/actions/paths-filter/action.yml

```diff
@@ -163,6 +163,15 @@ outputs:
         github.event_name == 'push' &&
         steps.filter-special-branch.outputs.stable-branch == 'true'
       }}
+  rdbms-integration-tests:
+    description: Output whether RDBMS integrations tests should be run based on code changes
+    value: >-
+      ${{
+        github.event_name == 'push' ||
+        github.event_name == 'pull_request' ||
+        steps.filter-common.outputs.github-actions-change == 'true' ||
```

Comment:
Since this job is relying on Maven, we also need the `steps.filter-common.outputs.maven-change == 'true' ||` condition here.